jwst/cube_build/ifu_cube.py

```diff
@@ -1343,6 +1352,7 @@ def map_fov_to_dqplane(self, this_par1, coord1, coord2, wave, roiw_ave, slice_no
 
                 iend = end_region
 
+                coord1_end = None
                 while iend > start_region:
                     index_use = np.where((wave_distance < roiw_ave) & (slice_no == iend))
```

Comment:
Similar question (different variables) as just above, regarding decrementing `iend` in the `while` loop.